tst/EnergyPlus/api/TestRuntime.py

```diff
@@ -79,6 +84,9 @@ def error_handler(severity: int, message: bytes) -> None:
 api.runtime.callback_begin_new_environment(state, environment_handler)
 api.runtime.callback_progress(state, progress_handler)
 api.functional.callback_error(state, error_handler)
+api.runtime.callback_end_system_sizing(state, common_callback_handler)
+api.runtime.callback_after_component_get_input(state, common_callback_handler)
+api.runtime.callback_unitary_system_sizing(state, common_callback_handler)
```

Comment:
These 3 lines fail if you try it in develop right now.  We properly pass two arguments from this `TestRuntime.py` script into the `callback_*` functions.  All good.  But then inside those `callback_*` functions, when they try to forward the arguments down into the guts of the C API, there is an argument mismatch as `ctypes` attempts to marshal the data because the functions were specified improperly.  Now that we properly tell `ctypes` what to expect with these function calls, everything is happy happy.